Tools/scripts/run_astyle.py

```diff
@@ -60,7 +60,7 @@ def check(self):
             self.progress("astyle check failed: (%s)" % (ret.stdout))
             self.retcode = 1
         if "Formatted" in ret.stdout:
-            self.progress("Files needing formatting found")
+            self.progress("Files needing formatting found. Please run ./Tools/scripts/run_astyle.py")
```

Comment:
This is a really strange thing to be printing in `run_astyle.py`!

If this is intended to make CI clearer, grepping the output of the dry-run might be a better idea.

Reply:
Is your expectation that we only print this in `build_ci.sh`?

Reply:
Yes, that's my expectation - don't confuse the user who is running this manually.

Reply:
Please look again. I've moved this message to build-ci.sh and check the error code.